addons/hr_attendance/security/hr_attendance_security.xml

```diff
@@ -62,7 +62,7 @@
             <field name="name">user: modify own attendance only</field>
             <field name="model_id" ref="model_hr_attendance"/>
             <field name="domain_force">[('employee_id.user_id','=',user.id)]</field>
-            <field name="perm_read" eval="0"/>
+            <field name="perm_read" eval="1"/>
```

Comment:
From a stability standpoint, it's ok to change this in stable, as it will only affect new databases.

From a privacy point of view, I think it's better to prevent `read` indeed, but as you can see with rule name, this was not the intended behavior initially. So you'll need to update the rule name too, and confirm this change functionally with a PO first.

Remember, we don't make changes in business logic just because someone opens a ticket, especially when there is no traceback/error. Because each customer has a different opinion on "the expected behavior", we need take our own decision, and stick to it  ;-)

Reply:
Thanks! I've changed the name.

> confirm this change functionally with a PO first

Indeed. FLL was OK with the change.

> Remember, we don't make changes in business logic just because someone opens a ticket, especially when there is no traceback/error. Because each customer has a different opinion on "the expected behavior", we need take our own decision, and stick to it ;-)

:+1: